README.md

```diff
@@ -0,0 +1,69 @@
+Overview
+========
+
+This document describes how to modify a [TripleO Queens OpenStack with Ceph Luminous Deployment](https://docs.openstack.org/tripleo-docs/latest/install/advanced_deployment/ceph_config.html) to include [Ceph Metrics](https://github.com/ceph/cephmetrics/wiki).
+
+TripleO's [composable networks](https://docs.openstack.org/tripleo-docs/latest/install/advanced_deployment/custom_networks.html) feature is used to deploy an overcloud with a dedicated CephMetrics network which is used by all Ceph nodes to share metrics data. Firewall ports are [opened](tht/cephmetrics.yaml) on all hosts with Ceph services for the [Prometheus exporter](http://docs.ceph.com/docs/mimic/mgr/prometheus). After the overcloud is deployed the overcloud admin configures the CephMetrics network as a [provider network](https://docs.openstack.org/newton/install-guide-rdo/launch-instance-networks-provider.html) and creates a separate [cephmetrics project](https://docs.openstack.org/keystone/queens/admin/cli-manage-projects-users-and-roles.html) with access to the provider network. Heat templates are [provided](instance) to launch an instance within the cephmetrics project and tools are provided to install ceph metrics with Ansible on the instance. An example of accessing the ceph metrics GUI via a floating IP is provided though it is at the discretion of the deployer how the ceph metrics GUI is accessed.
+
+![CephMetrics Logical Network Diagram](https://www.dropbox.com/s/qprrraef9hvvhzy/CephMetricsNetworkDiagram.png?raw=1)
+
+Deploying the Overcloud
+=======================
+
+There is a TripleO Heat Templates directory, [tht](tht), which contains the following templates.
+
+- [cephmetrics.yaml](tht/cephmetrics.yaml) opens the firewall ports
+- [network_data_ceph_metrics.yaml](tht/network_data_ceph_metrics.yaml) replaces the standard /usr/share/openstack-tripleo-heat-templates/network_data.yaml by adding a new CephMetrics network
+- [roles_data_ceph_metrics.yaml](tht/roles_data_ceph_metrics.yaml) replaces the standard /usr/share/openstack-tripleo-heat-templates/roles_data.yaml by redefining the Controller and CephStorage roles to include the CephMetrics network
+
+Create your own ~/templates directory and render your own ~/templates/environments/network-environment.yaml with the following:
+```
+rm -rf ~/templates 2> /dev/null
+cp -r /usr/share/openstack-tripleo-heat-templates ~/templates
+python ~/templates/tools/process-templates.py -n ~/tht/network_data_ceph_metrics.yaml -r ~/tht/roles_data_ceph_metrics.yaml -p ~/templates
```

Comment:
I see references to `~/tht` in a few places in this doc; should we be instructing the user to place them in that location, or should we update the paths in this PR?

Reply:
I also wonder if we should rename the template dir "heat-templates" or something else that's more obvious

Reply:
I don't think we should rename it and the paths in the rest of the document and the overcloud-deploy.sh are consistent with the ~/tht directory.

Reply:
We want ~/templates to be deletable rebuildable by the user as they tweak their undercloud while ~/tht is not something they'd probably change. We're just providing an example and they can vary it if they wish. I wanted it separate from ~/templates though.